### PR TITLE
Expose remote branch ref as an environment variable

### PR DESCRIPTION
**Pull request by surafelabebe**
The remote branch reference was exposed as an environment variable to be consumed by a pre-push hook, which makes git push decisions to remote branches with a certain name pattern.

### CODE REVIEW

File: pre_commit/commands/run.py [hunk 1]
```diff
@@ -371,6 +371,11 @@ def run(
         environ['PRE_COMMIT_FROM_REF'] = args.from_ref
         environ['PRE_COMMIT_TO_REF'] = args.to_ref
 
+    # Exposes remote branch ref as an environment variable
+    # for pre-push hooks consumption
+    if args.remote_branch:
+        environ['PRE_COMMIT_REMOTE_REF'] = args.remote_branch
```
Comment (asottile) [inline pre_commit/commands/run.py:377]: this should be combined with the block below -- the naming here is inconsistent as well

**Review by asottile — COMMENTED** [review]
hmmm I wonder if it's easy to extend the existing tests to cover this change

File: pre_commit/commands/run.py (hunk 1 above)
Comment (surafelabebe) [reply pre_commit/commands/run.py]: I updated the code as recommended.

**Review by asottile — APPROVED** [review]

**Comment by asottile** [comment]
seems I can't merge this because you made the pull request from a company account (this also seems to be why pre-commit.ci couldn't autofix your pull request):

```console
$ git push clumio-oss HEAD -f
ERROR: Permission to clumio-oss/pre-commit.git denied to asottile.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I'm going to close this and then reopen another pull request and maintain your authorship there

**Pull request closed without merging**